### ISSUE TITLE
Translation process description update

### ISSUE THREAD

**Issue by magul** [DjangoGirls/translation-guide#8]
Since I remember we do not follow onboarding of new translators described in a `README.md`.

I don't know if anybody actively monitors `translations@djangogirls.org`. I believe not, as @PluckyPrecious wrote:
> I’ve sent series of mail in regards to the resource below point_down , and got no response.

I also know, that:
 * you can freely join Crowdin project and become a translator, so there's no need to approve of any kind here,
 * we used to start new translations based on GitHub issues (for reference see: https://github.com/DjangoGirls/tutorial/issues/1656, https://github.com/DjangoGirls/tutorial/issues/1626, https://github.com/DjangoGirls/tutorial/issues/1507, https://github.com/DjangoGirls/tutorial/issues/1505, https://github.com/DjangoGirls/tutorial/issues/1479),
 * current unwritten policy (that I follow as self-claimed steward of Crowdin -> GitHub syncing) is that we/I publish non-proofread translations with `(beta)` mark (otherwise we would not have them so many).

The topic was started by @PluckyPrecious in https://github.com/DjangoGirls/tutorial/issues/1670.

CC: @PluckyPrecious @das-g

**Comment by RachellCalhoun** [comment]
Hiya @magul! I just wanted to pop in here to say that I check the translations email but probably not as often as I should. We get about 4 emails every few months at that address. So once a month or so I will go in and reply to all if we have any emails. Thanks for bringing it up! 

I would love to update the process and add some automation! When new people want to join the translation project, I check if the project exists, if it doesn't I create it and provide an invite link to join the project.

**Comment by magul** [comment]
Hey @RachellCalhoun, when you say
> I check if the project exists

what do you mean? Do you mean a new language on Crowdin? We can probably stick to the GitHub issues here, they will be a lot more transparent and will allow everybody, that manage Crowdin project to add target language.

**Comment by RachellCalhoun** [comment]
Yes, when someone emails asking to join a project that is what I have done. I check on crowdin to see if the translation project for that language exists.

I agree transparency is better on github. I think this should probably be part of the larger conversation about the process for onboarding new people.

**Comment by PluckyPrecious** [comment]
Hi @magul, thanks for helping out to resolve this. 🤝 

Hi @RachellCalhoun, Just got your response to the mail thanks. 🤗
